### PR TITLE
Requirements Section for Bug Triage

### PR DESCRIPTION
**Pull request by soggiest**
This PR adds a Requirements section to the Bug Triage handbook and fixes a minor spelling change in the CI Signal handbook.

### CODE REVIEW

File: release-team/role-handbooks/bug-triage/README.md [hunk 1]
```diff
@@ -43,6 +43,38 @@ There are four relevant periods where your workload changes:
...
+
+Additionally, the following qualifications make a candidate more suitable for the Bug Triage team, but are not requirements:
+
+- Experience with quality assurance and bug tracking systems.
+
+### Additional Requirements for Leads
+ 
+In addition to the above requirements for Shadows, most of which become prerequisites, Bug Triage Leads must:
+
+- Have the ability to add a milestone to issues, so must be a member of the [milestone maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers)
```
Comment (justaugustus) [inline release-team/role-handbooks/bug-triage/README.md]:
```suggestion
- Have the ability to add a milestone to issues, so must be a member of the [milestone maintainers](/release-team/README.md#milestone-maintainers)
```

File: release-team/role-handbooks/bug-triage/README.md [hunk 2]
```diff
@@ -43,6 +43,38 @@ There are four relevant periods where your workload changes:
...
+- Have a working knowledge of GitHub labelling and API. Bug Triage leads must be able to identify relevant issues and PRs for a given milestone.
```
Comment (justaugustus) [inline release-team/role-handbooks/bug-triage/README.md]:
```suggestion
- Have a working knowledge of GitHub labeling. Bug Triage leads must be able to identify relevant issues and PRs for a given milestone.
```

File: release-team/role-handbooks/bug-triage/README.md [hunk 3]
```diff
@@ -43,6 +43,38 @@ There are four relevant periods where your workload changes:
...
+- Have an understanding of what defines a "Release Blocking" issue or PR, or know who to contact to determine that information.
```
Comment (justaugustus) [inline release-team/role-handbooks/bug-triage/README.md]:
```suggestion
- Have an understanding of what defines a ["Release Blocking"](/release-blocking-jobs.md) issue or PR, or know who to contact to determine that information.
```

File: release-team/role-handbooks/bug-triage/README.md [hunk 4]
```diff
@@ -43,6 +43,38 @@ There are four relevant periods where your workload changes:
...
+
+### Additional Requirements for Shadows
+
+The following requirements are all things that Leads should already have, and Shadows should acquire, with coaching, within the first weeks of the release cycle:
+
+- Have signed the contributor CLA for Kubernetes.
+- Become a Kubernetes org member.  In many cases, this will be done with the sponsorship of the Bug Triage Lead or Release Lead in the first week of the cycle.
+  - The process to become one of these is in [our community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-for-outside-collaborators)
+- General familiarity with GitHub labels and how to find issues/PRs for the current milestone. 
+- Willingness and ability to follow-up with contributors about issues/PRs, on Slack, email, Discuss, and SIG meetings, as appropriate.
```
[the remark this answers is not in the document]
Comment (soggiest) [reply release-team/role-handbooks/bug-triage/README.md]: Which of these are you referring to?

**Comment by justaugustus** [comment]
@soggiest -- can you squash your commits? See my comment here: https://github.com/kubernetes/sig-release/pull/666#issuecomment-502406663

File: release-team/role-handbooks/bug-triage/README.md [hunk 5]
```diff
@@ -43,6 +43,38 @@ There are four relevant periods where your workload changes:
...
+- General knowledge of our SIG system and SIGs' areas of responsibility.
```
[the remark this answers is not in the document]
Comment (soggiest) [reply release-team/role-handbooks/bug-triage/README.md]: I'm not sure where we can stick that in, want to add it in a subsequent PR?

File: release-team/role-handbooks/bug-triage/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (soggiest) [reply release-team/role-handbooks/bug-triage/README.md]: I think Stephen added some information around that

**Comment by soggiest** [comment]
> @soggiest -- can you squash your commits? See my comment here: [#666 (comment)](https://github.com/kubernetes/sig-release/pull/666#issuecomment-502406663)

Just squashed

**Comment by justaugustus** [comment]
Awesome!
/lgtm
/approved

**Pull request merged**